code/Model/Submission/SubmittedFileField.php

```diff
@@ -41,14 +41,24 @@ public function getFormattedValue()
         $name = $this->getFileName();
         $link = $this->getLink();
```

Comment:
```suggestion
        $link = $this->getLink(false);
```

Getting the Link automatically session grant you the file, even if it's not outputted on the page.

Reply:
But method `getLink()` doesn't take any arguments?! https://github.com/silverstripe/silverstripe-userforms/blob/ff88871e709351258a5a81b909d4f8bf6cf69f45/code/Model/Submission/SubmittedFileField.php#L82

Reply:
Sorry ... that was a brain fart on my part. 

You might need to update the `getLink` method to call `$file->getURL(false);`. I don't think you need an absolute URL here. If you do, you might need to replicate part of `File::getAbsoluteURL()` in this class.

That should fix the new unit test we just added.

Reply:
DONE